MyNewFile.txt

```diff
@@ -1,2 +1,3 @@
 Something
+Test
```

Comment:
Are you sure this is what you need to change.

Reply:
Thank you, I've added the line.